docs/manual/working/scalaGuide/main/sql/ScalaAnorm.md

```diff
@@ -639,6 +639,17 @@ val result: List[(String, Int)] =
   SQL("select * from Country").as(parser.flatten.*)
 ```
 
+A `RowParser` can be combined with any function to applied it with extracted columns.
+
+```scala
+import anorm.SqlParser.{ int, str, to }
+
+def display(name: String, population: Int): String = 
+  s"The population in $name is of $population."
+
+val parser = str("name") ~ int("population") map (to(display _))
```

Comment:
Note that `display` function must be partially applied with `_` to be used with `to` (cannot write `to(display)`).